src/TestColoursFonts.tsx

```diff
@@ -0,0 +1,36 @@
+﻿import React from 'react';
+import  './TestColoursFonts.scss';
+
+export function TestColoursFonts() {
+    return (
+        <div>
+          <br/>
+            <br/>
+            <br/>
+            
+            <br/>
+            <br/>
+            <br/>
+            <h1>Header One  HEADER ONE</h1>
+            <h2>Header Two HEADER TWO</h2>
+            <h3>Header Three HEADER THREE</h3>
+         
+            <p>Hello World! I am a paragraph.</p>
+            <p className="labelClass"> DAY
+                <br/>
+            <input type="text"/>
+            <br/>
+            MONTH
+                <br/>
+            <input type="text"/>
+            <br/>
+            YEAR
+                <br/>
+            <input type="text"/></p>
+            <br/>
+            <p className="quoteClass">Mars tugs at the human imagination like no other planet.<br/>
+            With a force mightier than gravity, it attracts the eye to the shimmering red presence in the clear night sky.</p>
+            <img src="https://coolors.co/export/png/241e4e-960200-ce6c47-f4a261-eadaa2"/>
+        </div>
+    );
+}
```

Comment:
This is a really nice idea, but we probably don't want to actually deploy it (as it isn't really a page that we'd want anyone to see)

Do you think you could apply the styles you have made to a few of the actual components?